src/unity/python/doc/source/turicreate.visualization.rst

```diff
@@ -4,14 +4,24 @@ Visualization
 
 .. currentmodule:: turicreate
 
-Visualize data in various forms including scatter plots, heat maps, box and
-whisker plots.
+Visualize data in various forms including scatter plots, heat maps, categorical heatmaps, box and whisker plots, columnwise summaries, histograms, and item frequency plots.
 
 .. autosummary::
   :toctree: generated/
   :nosignatures:
 
  show
  SArray.show
+ SArray.plot
+ SArray.explore
  SFrame.show
+ SFrame.plot
  SFrame.explore
+ visualization.scatter
```

Comment:
I'm not sure how these links will render or what they will link to. We should try rendering this page as HTML and see what it looks like.

You can run `make html` from `src/unity/python/doc` (maybe `source` in there, I don't remember), while in the development python env, and see how it looks.